invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/api/DepositFilesService.js

```diff
@@ -169,7 +168,7 @@ export class RDMDepositFilesService extends DepositFilesService {
         fileData.links
       );
     } catch (error) {
-      await this.delete(initializedFileMetadata.links);
+      console.error(error);
```

Comment:
comment: probably leftover :)

Reply:
this was on purpose, I want to display the errors in the console